### ISSUE TITLE
[Feature] Add icon

### ISSUE THREAD

**Issue by gabrc52** [H-M-H/Weylus#53]
The `input-tablet` icon would be a good fit for this app, which could be added as a potential taskbar/dock icon, and for a potential application menu entry if packaged.

Adwaita:

![image](https://user-images.githubusercontent.com/24363938/100279932-0773de80-2f2d-11eb-8956-db44e04f946d.png)

Papirus: 

![image](https://user-images.githubusercontent.com/24363938/100280146-53bf1e80-2f2d-11eb-8da9-77a28430cfce.png)

**Comment by H-M-H** [comment]
I agree, an icon and a .desktop file should be added! Looks like the specs for .desktop files can be found here: https://freedesktop.org/wiki/Specifications/desktop-entry-spec/

I don't really have time to do this right now but pull requests are welcomed! Additionally maybe someone wants to design a custom icon for Weylus.

**Comment by H-M-H** [comment]
Update: I added a .desktop file specifying `input-tablet` as icon a while ago. A custom icon for Weylus would be the next step.

**Comment by creepertron95** [comment]
Icon design proposal:

![Weylus](https://user-images.githubusercontent.com/53346722/125203346-a2f85c00-e23d-11eb-9074-efe609adc098.png)

---

[weylus.zip](https://github.com/H-M-H/Weylus/files/6797286/weylus.zip)

**Comment by 1sra3l** [comment]
Hi, it might be wise to note that if someone does not have their program in the system's `$PATH` your desktop file will not work.
I am not sure if Rust does something like weylus.desktop.in but you could potentially set the Desktop file's `Path=` to the binary install location.  I am rather new to Rust, but not new to the XDG specs

**Comment by H-M-H** [comment]
@creepertron95 Thank you very much for your design proposal! This is definitely better than anything I could produce! Some thoughts on the design:

- It looks very clean but also pretty generic, I think it would be better to have something more recognizable.
- There is no hint at stylus or touch input which I think should be included somehow, if the icon tries to describe what Weylus does. But to be fair, I think this is pretty hard to do especially considering that things should stay recognizable even at small size. That's why perhaps something more abstract might be better here.

Thanks again for your work!

@1sra3l Thanks for the hint! Currently this should only be a problem if you either build Weylus yourself or download the linux.zip. I am not sure how to go about this though, as there is no installation script that could know where the binary ends up.

**Comment by tinywrkb** [comment]
Note that Flatpak requires apps to ship their own icons, so using one from the icon theme will block a possible Flathub submission.
